tests/unit/testResources/environments.ts

```diff
@@ -3,28 +3,32 @@ import {
   DirectEnvironment,
   LocalEnvironment,
 } from "../../../src/models/environment";
+import { EnvironmentId } from "../../../src/models/resource";
 import { TEST_DIRECT_CONNECTION_ID, TEST_LOCAL_CONNECTION } from "./connection";
 
+export const TEST_CCLOUD_ENVIRONMENT_ID = "env-abc123" as EnvironmentId;
```

Comment:
Define the constant ID first (and using `EnvironmentId` type), then express the constant objects in terms of the constant IDs.